docs/src/main/asciidoc/jgit.adoc

```diff
@@ -52,5 +52,5 @@ Here is an example using it in a JAX-RS endpoint:
 
 [WARNING]
 ====
-When running in native mode, make sure that the link:native-and-ssl#the-sunec-library-and-friends[SSL access is configured correctly].
+When running in native mode, make sure that the link:native-and-ssl#the-truststore-path[SSL access is configured correctly].
```

Comment:
Good catch, thanks!

Reply:
Maybe we can add an explicit link instead of using the generated one? WDYT? @ahus1

Reply:
I've added the explicit link to `native-and-ssl.adoc`. I also found some old SunEC content in native-and-ssl.adoc that I removed.

Reply:
Added explicit link in native-and-ssl.adoc. I also found some SunEC content in native-and-ssl.adoc that I removed. Please double-check.